justfile

```diff
@@ -224,6 +224,15 @@ compile-ios target:
     cd "$RUST_WORKSPACE"; \
     cargo build --target {{target}} -p xayn-discovery-engine-bindings --release
 
+# Compiles the bindings for the iphoneos (aarch64) and iphonesimulator (x86_64)
+# and copies the binaries to the flutter project
+compile-ios-local:
+    #!/usr/bin/env sh
```

Comment:
Should-have: I just realized way may want to `set -eu` here, especially so that the command doesn't keep running if one of the commands fails.

Reply:
done